Makefile

```diff
@@ -190,9 +190,14 @@ test-markdown test/markdown:
 test-sanity test/sanity: tidy build/operator-sdk lint
 	./hack/tests/sanity-check.sh
 
-TEST_PKGS:=$(shell go list ./... | grep -v -P 'github.com/operator-framework/operator-sdk/(hack/|test/)')
+# NOTE: It is required to ignore the : pkg/ansible/proxy/proxy_test.go which requires network permissions
+TEST_PKGS:=$(shell go list ./... | grep -v -P 'github.com/operator-framework/operator-sdk/(hack/|test/|pkg/ansible/proxy/)')
 test-unit test/unit: ## Run the unit tests
-	$(Q)go test -coverprofile=coverage.out -covermode=count -count=1 -short ${TEST_PKGS}
+	# -failfast: disables running additional tests after any test fails
+	# -tags=integration: run tests tagged as integration
+	# -coverprofile: define the name of the file with the coverage
+	# -covermode: will count the % of coverage
+	$(Q)go test -failfast -tags=integration -coverprofile=coverage.out -covermode=count ${TEST_PKGS}
```

Comment:
@jmrodri if we do not use the sort the % is increased since more tests will be executed. See: `COVERAGE INCREASED (+5.8%) TO 47.901%`. However, the `pkg/ansible/proxy/proxy_go` cannot be executed because of it requires some network permissions.

Reply:
Thank you for your help and let me know of the `/` and the end :-)